vowpalwabbit/conditional_contextual_bandit.cc

```diff
@@ -184,40 +225,29 @@ void remove_slot_features(example* shared, example* slot)
   }
 }
 
-// Generates all combinations of 4th order interactions between namespaces in [shared,history,slot,action]
+// Generates interaction between all namespaces and interactions with the slot id namespace.
```

Comment:
The comment is wrong?

Reply:
My comment is clumsy, struggled to word it in a way that makes sense.

"Generates quadratics between each namespace and the slot id as well as appends slot id to every existing interaction"?

Reply:
Sounds good.